javascriptv3/example_code/cross-services/lambda-scheduled-events/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "lambda-scheduled-events-example",
+  "version": "1.0.0",
+  "description": "Using the AWS SDK for JavaScript V3 to build and deploy Lambda functions using scheduled events.",
+  "main": "index.js",
+  "repository": "git@github.com/awsdocs/aws-doc-sdk-examples/tree/master/javascriptv3/example_code/cross-services/lambda-scheduled-events.git",
+  "author": "brmur",
+  "license": "MIT",
```

Comment:
Our repo is Apache 2.0 by default, did you mean to use MIT?

Reply:
done